index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="href='https://fonts.googleapis.com/css?family=Poppins" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+</head>
+
+<body>
+
+
+    <header>
+        <nav class="topnav">
+            <h2 class="logo">My Logo</h2>
+
+            <a class="icon">
+                <i class="fa fa-bars"></i>
+            </a>
+
+        </nav>
+    </header>
+    <section class="headline"></section>
+
+    <h1 class="name">
+        I'm Oluwatosin Festus Ajibola
+        Glad to meet you!
+    </h1>
+
+    <p class="words">
+        I'm a software developer! I can help you build a product,
+        feature or website. Look through some of my work and experience!
+        If you like what you see and have a project need coded, dont hesitate
+        to contact me.
+    </p>
+
+
+    <p class="connect">LET'S CONNECT!</p>
+
+
+    <ul class="social-media">
+        <li><a href="#"><img src="./images/twitter.svg" alt="twitter icon"></a></li>
+        <li><a href="#"><img src="./images/Linkedin.svg" alt="Linkedin icon"></a></li>
+        <li><a href="#"><img src="./images/Mirror.svg" alt="Mirror icon"></a></li>
+        <li><a href="#"><img src="./images/Github.svg" alt=""></a></li>
+        <li><a href="#"><img src="./images/peace.svg" alt="peace icon"></a></li>
+
+    </ul>
+
+
+
+    </section>
+
+
+
```

Comment:
## for HTML Best Practices ♻️ 

- [ ] Kindly it would be better to reduce empty lines in your codes as Removing unnecessary empty lines in HTML code can help to reduce the file size of the page, which can lead to faster page load times and a better user experience. It also makes the code more compact and easier to read, which can be helpful for developers who are working on the code. 😎